### ISSUE TITLE
Adding CategoricalFocalLoss() to utils

### ISSUE THREAD

**Issue by gireeshkbogu** [qubvel/segmentation_models.pytorch#170]
In Keras total loss calculated as follows for a multi-class segmentation problem. 
```
# Segmentation models losses can be combined together by '+' and scaled by integer or float factor
# set class weights for dice_loss (car: 1.; pedestrian: 2.; background: 0.5;)
dice_loss = sm.losses.DiceLoss(class_weights=np.array([1, 2, 0.5])) 
focal_loss = sm.losses.BinaryFocalLoss() if n_classes == 1 else sm.losses.CategoricalFocalLoss()
total_loss = dice_loss + (1 * focal_loss)
```
However, I can not find CategoricalFocalLoss() in PyTorch implementation. Could you please update this? It is currently a roadblock for many users here. Also is there a way you can use class_weights in dice_loss like in Keras? Many thanks.

**Comment by zhongqiu1245** [comment]
I have the same problem

**Comment by albert-92** [comment]
any updates on this one?
